isthmus/src/main/java/io/substrait/isthmus/expression/ExpressionRexConverter.java

```diff
@@ -213,6 +213,12 @@ public RexNode visit(FieldReference expr) throws RuntimeException {
     return visitFallback(expr);
   }
 
+  @Override
+  public RexNode visit(Expression.Window expr) throws RuntimeException {
+    // todo:to construct the RexOver
```

Comment:
If Substrait -> Rex conversion is not implemented yet, better not to Override this method. In stead, let it call `visitFallBack`.

Reply:
good catch, done.